tests/EditableFormFieldTest.php

```diff
@@ -217,4 +217,16 @@ public function testFormatDisplayRules()
         // The opposite method should be to return it to its original state, i.e. show it again
         $this->assertSame('removeClass("hide")', $displayRules['opposite']);
     }
+
+    /**
+     * Tests that certain elements are rendered with custom templates
+     */
+    public function testCustomTemplates()
+    {
+        $radio = $this->objFromFixture('EditableRadioField', 'radio-field');
+        $this->assertEquals('UserFormsOptionSetField', $radio->getFormField()->getTemplate());
+
+        $checkboxGroup = $this->objFromFixture('EditableCheckboxGroupField', 'checkbox-group');
+        $this->assertEquals('UserFormsCheckboxSetField', $checkboxGroup->getFormField()->getTemplate());
```

Comment:
Can you put these as separate tests into a test class for each of the relevant fields?